ocf_datapipes/load/gsp.py

```diff
@@ -0,0 +1,103 @@
+import datetime
+
+import geopandas as gpd
+import numpy as np
+import pandas as pd
+import xarray as xr
+from torchdata.datapipes import functional_datapipe
+from torchdata.datapipes.iter import IterDataPipe
+
+
+@functional_datapipe("open_gsp")
+class OpenGSPIterDataPipe(IterDataPipe):
+    def __init__(
+        self,
+        gsp_pv_power_zarr_path: str,
+        gsp_id_to_region_id_filename: str,
+        sheffield_solar_region_path: str,
```

Comment:
althought it made by solar sheffiled, i wonder if we should just rename it as gsp_region_path, as NG-ESO publish it

Reply:
or gsp_shapes_path